### PR TITLE
Add Bulgarian translations

### PR DESCRIPTION
**Pull request by erogleva**

### CODE REVIEW

File: phonenumber_field/locale/bg/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Bulgarian translations
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# Elena Rogleva <elena.rogleva@gmail.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
```
Comment (francoisfreitag) [inline phonenumber_field/locale/bg/LC_MESSAGES/django.po:8]: Other locales don’t provide that information.
```suggestion
"Project-Id-Version: \n"
```

File: phonenumber_field/locale/bg/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+msgid "Phone number"
+msgstr "Телефонен номер"
+
+#: serializerfields.py:9
+msgid "Enter a valid phone number."
+msgstr "Моля, въведете валиден телефонен номер."
+
+#: validators.py:11
+msgid "The phone number entered is not valid."
+msgstr "Въведеният телефонен номер е невалиден."
```
Comment (francoisfreitag) [inline phonenumber_field/locale/bg/LC_MESSAGES/django.po:44]: Please add a newline at the end of the file.

**Review by francoisfreitag — APPROVED** [review]
Looking good, thanks!

**Comment by francoisfreitag** [comment]
Thanks for updating!

**Comment by francoisfreitag** [comment]
I added the compiled messages with `$ django-admin compilemessages --locale bg`.
`

**Comment by erogleva** [comment]
> I added the compiled messages with `$ django-admin compilemessages --locale bg`.

Great, thanks also for the review!

**Pull request merged**